3.6
An error occurred while executing the following cell:
------------------
import matplotlib.pyplot as plt
import seaborn as sns

% matplotlib inline
------------------


UsageError: Line magic function `%` not found.

Executability                            : False
